std/bigint.d

```diff
@@ -1019,6 +1019,53 @@ public:
         assert(aa[BigInt(456)] == "def");
     }
 
+    /**
+     * Gets the nth number in the underlying representation that makes up the whole
+     * `BigInt`.
+     *
+     * Params:
+     *     T = the type to view the underlying representation as
+     *     n = The nth number to retrieve. Must be less than $(LREF ulongLength) or
+     *     $(LREF uintLength) with respect to `T`.
+     * Returns:
+     *     The nth `ulong` in the representation of this `BigInt`.
+     */
+    T getDigit(T = ulong)(size_t n) const
+        if (is(T == ulong) || is(T == uint))
```

Comment:
Constraint indentation should be on the same level... I thought there was a test for this :/

Reply:
https://github.com/dlang-community/D-Scanner/pull/450 - it just never made it into DScanner.